app/controllers/admin/users/list.js

```diff
@@ -1,108 +1,134 @@
-      template         : 'components/ui-table/cell/admin/users/cell-user-links',
-      disableSorting   : true,
-      disableFiltering : true
-    },
-    {
-      propertyName : 'created-at',
-      title        : 'Member Since',
-      template     : 'components/ui-table/cell/admin/users/cell-created-at'
-    },
-    {
-      propertyName     : 'lastAccessedAt',
-      title            : 'Last Accessed',
-      template         : 'components/ui-table/cell/cell-simple-date',
-      dateFormat       : 'MMMM DD, YYYY - hh:mm A',
-      disableSorting   : true,
-      disableFiltering : true
-    },
-    {
-      template         : 'components/ui-table/cell/admin/users/cell-actions',
-      title            : 'Actions',
-      disableSorting   : true,
-      disableFiltering : true
+import { computed, action } from '@ember/object';
+import { or } from '@ember/object/computed';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
+
+
+export default class extends Controller.extend(EmberTableControllerMixin) {
+  @or('authManager.currentUser.isSuperAdmin', 'authManager.currentUser.isAdmin') hasRestorePrivileges;
+
+  @computed()
+  get columns() {
+    return [
+      {
+        name          : 'Name',
+        valuePath     : 'firstName',
+        cellComponent : 'ui-table/cell/admin/users/cell-first-name',
+        options       : {
+          hasRestorePrivileges: this.hasRestorePrivileges
+        },
+        actions: {
+          moveToUserDetails : this.moveToUserDetails.bind(this),
+          deleteUser        : this.deleteUser.bind(this),
+          openEditModal     : this.openEditModal.bind(this),
+          restoreUser       : this.restoreUser.bind(this)
+        }
+
+      },
+      {
+        name      : 'Email',
+        valuePath : 'email'
+
+
+      },
+      {
+        name          : 'Status',
+        valuePath     : 'status',
+        cellComponent : 'ui-table/cell/admin/users/cell-status'
+
+
+      },
+      {
+        name          : 'System Roles',
+        valuePath     : 'systemRoles',
+        cellComponent : 'ui-table/cell/admin/users/cell-system-roles'
+
+
+      },
+      {
+        name          : 'Event Roles',
+        valuePath     : 'eventRoles',
+        cellComponent : 'ui-table/cell/admin/users/cell-event-roles'
+
+
+      },
+      {
+        name          : 'User Links',
+        valuePath     : 'userLinks',
+        cellComponent : 'ui-table/cell/admin/users/cell-user-links'
+
+
+      },
+      {
+        name          : 'Member Since',
+        valuePath     : 'createdAt',
+        cellComponent : 'ui-table/cell/admin/users/cell-created-at'
+      },
+      {
+        name          : 'Last Accessed',
+        valuePath     : 'lastAccessedAt',
+        cellComponent : 'ui-table/cell/cell-simple-date',
+        options       : {
+          dateFormat: 'MMMM DD, YYYY - hh:mm A'
+        }
+
+
+      }
+    ];
+  }
+
+  @computed('model.[]')
+  get rows() {
```

Comment:
Use model directly. One less computed property. One less iteration of the dataset.